ivy/data_classes/container/base.py

```diff
@@ -4005,7 +4005,7 @@ def _add_newline(str_in):
         return new_dict
 
     def __dir__(self):
-        return list(super.__dir__(self)) + list(self.keys())
+        return list(super().__dir__(self)) + list(self.keys())
```

Comment:
You dont pass in self when using `super()`

Reply:
@sherry30 sir, I made the requested changes. Can you please check now, Thank you.